From d4cbabf21a730acf454035f512fa00134471e185 Mon Sep 17 00:00:00 2001
From: Vlad Pelin <vlad.pelin@nxp.com>
Date: Mon, 11 Oct 2021 17:10:42 +0300
Subject: [PATCH 3/5] pkcs: renamed hse-usr.h to libhse.h

renamed hse-usr.h -> libhse.h to better reflect its need
for inclusion to use libhse.so

Issue: ALB-7684
Upstream-Status: Pending 

Signed-off-by: Vlad Pelin <vlad.pelin@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 libhse/hse-usr.c               | 2 +-
 libhse/{hse-usr.h => libhse.h} | 0
 libpkcs/pkcs11_context.h       | 3 ++-
 3 files changed, 3 insertions(+), 2 deletions(-)
 rename libhse/{hse-usr.h => libhse.h} (100%)

diff --git a/libhse/hse-usr.c b/libhse/hse-usr.c
index 074987d..2c15e2f 100644
--- a/libhse/hse-usr.c
+++ b/libhse/hse-usr.c
@@ -15,7 +15,7 @@
 #include <unistd.h>
 #include <errno.h>
 
-#include "hse-usr.h"
+#include "libhse.h"
 
 #include "hse_interface.h"
 
diff --git a/libhse/hse-usr.h b/libhse/libhse.h
similarity index 100%
rename from libhse/hse-usr.h
rename to libhse/libhse.h
diff --git a/libpkcs/pkcs11_context.h b/libpkcs/pkcs11_context.h
index 2ec6b2c..ffbfbba 100644
--- a/libpkcs/pkcs11_context.h
+++ b/libpkcs/pkcs11_context.h
@@ -6,9 +6,10 @@
 #ifndef ___PKCS11_CONTEXT_H___
 #define ___PKCS11_CONTEXT_H___
 
+#include <stdint.h>
+#include <libhse.h>
 #include "pkcs11.h"
 #include "simclist.h"
-#include "hse-usr.h"
 #include "hse_interface.h"
 
 #define LIBRARY_VERSION_MAJOR 0u
-- 
2.17.1

